apps/widget-e2e/src/describer/modelAssessment/IModelAssessmentData.ts

```diff
@@ -117,6 +117,6 @@ export interface IWhatIfCounterfactualsData {
 
 export enum RAINotebookNames {
   "ClassificationModelAssessment" = "responsibleaidashboard-census-classification-model-debugging.py",
-  "RegressionDecisionMaking" = "responsibleaidashboard-diabetes-decision-making.py",
-  "RegressionModelAssessment" = "responsibleaidashboard-diabetes-regression-model-debugging.py"
+  "DiabetesRegressionModelDebugging" = "responsibleaidashboard-diabetes-regression-model-debugging.py",
+  "HousingClassificationModelDebugging" = "responsibleaidashboard-housing-classification-model-debugging.py"
```

Comment:
Is this only testing the regression scenario for aggregate feature importance? What about the classification scenario?

Reply:
Yes this PR is only for aggregate feature importance scenario. Other scenarios will be covered in coming PRs.

Reply:
Seems like you are adding this test case for regression task only. What about classification task?

Reply:
I didn't quite get this. Is it different notebook? or content in dashboard I missed?

Reply:
If you are talking about notebook https://github.com/microsoft/responsible-ai-toolbox/blob/main/notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb, this will be done in later tasks. I am covering notebooks one by one. Eventually we will have e2e tests for all notebooks.

Reply:
I meant was this test seems to only apply to regression notebook diabetes dataset. That's why I was wondering why for some classification scenario this test did not apply. But if you are tracking this at your end to add it for classification, I guess this is fine.